packages/grafana-schema/src/schema/mudball.gen.ts

```diff
@@ -489,33 +489,54 @@ export enum BarGaugeDisplayMode {
 }
 
 /**
- * Interface for table cell types that have no additional options.
+ * / Auto mode table cell options
  */
 export interface TableAutoCellOptions {
-  type: TableCellDisplayMode;
```

Comment:
this is the source of the type issues, because this allowed all enum values for TableCellDisplayMode it completely breaks the union type and Typescripts ability to use this property as a descriminator

Reply:
Ah I see, I definitely misunderstood how that would work in practice and it seemed like this _should_ work. That's clearly not the case though 🤦